cmd/archeio/main.go

```diff
@@ -42,13 +40,18 @@ func main() {
 	port := getEnv("PORT", "8080")
 
 	// make it possible to override k8s.gcr.io without rebuilding in the future
-	upstreamRegistry := getEnv("UPSTREAM_REGISTRY", defaultUpstreamRegistry)
+	registryConfig := app.RegistryConfig{
+		UpstreamRegistryEndpoint: getEnv("UPSTREAM_REGISTRY_ENDPOINT", "https://k8s.gcr.io"),
+		UpstreamRegistryPath:     getEnv("UPSTREAM_REGISTRY_PATH", ""),
+		InfoURL:                  "https://github.com/kubernetes/k8s.io/tree/main/registry.k8s.io",
```

Comment:
this accidentally reverts https://github.com/kubernetes-sigs/oci-proxy/pull/85, reopening https://github.com/kubernetes-sigs/oci-proxy/issues/83